{"uuid": "c903f694-bb03-4c50-b1a1-07260dd1b32f", "children": ["fa7b3d0d-3c86-48e9-9708-4e8566b1c04a"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=128.0.6613.138)\nStacktrace:\n\tGetHandleVerifier [0x00007FF6E01E9412+29090]\n\t(No symbol) [0x00007FF6E015E239]\n\t(No symbol) [0x00007FF6E001B1DA]\n\t(No symbol) [0x00007FF6DFFEFAF5]\n\t(No symbol) [0x00007FF6E009E2C7]\n\t(No symbol) [0x00007FF6E00B5EB1]\n\t(No symbol) [0x00007FF6E0096493]\n\t(No symbol) [0x00007FF6E00609D1]\n\t(No symbol) [0x00007FF6E0061B31]\n\tGetHandleVerifier [0x00007FF6E050871D+3302573]\n\tGetHandleVerifier [0x00007FF6E0554243+3612627]\n\tGetHandleVerifier [0x00007FF6E054A417+3572135]\n\tGetHandleVerifier [0x00007FF6E02A5EB6+801862]\n\t(No symbol) [0x00007FF6E016945F]\n\t(No symbol) [0x00007FF6E0164FB4]\n\t(No symbol) [0x00007FF6E0165140]\n\t(No symbol) [0x00007FF6E015461F]\n\tBaseThreadInitThunk [0x00007FFAF746257D+29]\n\tRtlUserThreadStart [0x00007FFAF8B4AF28+40]\n\n", "trace": "  File \"C:\\Users\\Msi\\PycharmProjects\\KalbelaAutomation\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\Msi\\PycharmProjects\\KalbelaAutomation\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Msi\\PycharmProjects\\KalbelaAutomation\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Msi\\PycharmProjects\\KalbelaAutomation\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Msi\\PycharmProjects\\KalbelaAutomation\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Msi\\PycharmProjects\\KalbelaAutomation\\tests\\test_top_section.py\", line 18, in setup\n    driver.maximize_window()\n  File \"C:\\Users\\Msi\\PycharmProjects\\KalbelaAutomation\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 506, in maximize_window\n    self.execute(command, None)\n  File \"C:\\Users\\Msi\\PycharmProjects\\KalbelaAutomation\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Msi\\PycharmProjects\\KalbelaAutomation\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1726636232038, "stop": 1726636235711}], "start": 1726636232038, "stop": 1726636235812}